libraries/c_sdk/standard/ble/include/iot_ble_data_transfer.h

```diff
@@ -44,7 +44,8 @@ typedef enum IotBleDataTransferChannelEvent
     IOT_BLE_DATA_TRANSFER_CHANNEL_OPENED = 0,    /**< Indicates if the channel is opened and ready to read or write data. */
     IOT_BLE_DATA_TRANSFER_CHANNEL_DATA_RECEIVED, /**< Event invoked when the last chunk of a large object or a small packet is received on the channel. */
     IOT_BLE_DATA_TRANSFER_CHANNEL_DATA_SENT,     /**< Event invoked after the last chunk of data stream is sent over the channel. */
-    IOT_BLE_DATA_TRANSFER_CHANNEL_CLOSED         /**< Event invoked when the channel is closed. */
+    IOT_BLE_DATA_TRANSFER_CHANNEL_CLOSED,        /**< Event invoked when the channel is closed. */
+    IOT_BLE_DATA_TRANSFER_CHANNEL_ERROR_UPDATED  /**< Event invoked when error status bits is overwritten */
```

Comment:
Could this be renamed as `IOT_BLE_DATA_TRANSFER_CHANNEL_PEER_ERROR` to confirm with the API name?

Reply:
added in latest commit